src/torrent.cpp

```diff
@@ -3175,7 +3175,8 @@ bool is_downloading_state(int const st)
 
 				aep.updating = true;
 				aep.next_announce = now;
-				aep.min_announce = now;
+				aep.min_announce = now
+					+ settings().get_int(settings_pack::min_announce_interval);
```

Comment:
This is causing the build to fail.
`libtorrent/src/torrent.cpp: In member function ‘void libtorrent::torrent::announce_with_tracker(uint8_t)’:
libtorrent/src/torrent.cpp:3179:41: error: no match for ‘operator+’ (operand types are ‘const time_point32’ {aka ‘const std::chrono::time_point<std::chrono::_V2::system_clock, std::chrono::duration<int> >’} and ‘int’)
 3178 |                                 aep.min_announce = now
      |                                                    ~~~
      |                                                    |
      |                                                    const time_point32 {aka const std::chrono::time_point<std::chrono::_V2::system_clock, std::chrono::duration<int> >}
 3179 |                                         + settings().get_int(settings_pack::min_announce_interval);
      |                                         ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                             |
      |                                                             int
`